pandas/compat/numpy/function.py

```diff
@@ -222,7 +222,7 @@ def validate_cum_func_with_skipna(skipna, args, kwargs, name) -> bool:
         skipna = True
 
     validate_cum_func(args, kwargs, fname=name)
-    return skipna
+    return bool(skipna)
```

Comment:
I'm not really a fan of explicitly creating new objects like this to appease typing; I think an ignore is preferable. Any idea how this gets out of whack in the first place?

Reply:
> I'm not really a fan of explicitly creating new objects like this to appease typing; I think an ignore is preferable

works for me

> Any idea how this gets out of whack in the first place?

The return annotation wants bool but we could have np.bool_

Reply:
Gotcha - yea I would just ignore for now then